docs/recipes/github-actions.md

```diff
@@ -0,0 +1,96 @@
+# Using semantic-release with [GitHub Actions](https://help.github.com/en/categories/automating-your-workflow-with-github-actions)
+
+## Environment variables
+
+The [Authentication](../usage/ci-configuration.md#authentication) environment variables can be configured with [Secret variables](https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables).
+
+For this example, you'll need to setup a `NPM_TOKEN` and a `GH_TOKEN`, this last one is recommended to use [`@semantic-release/git`](https://github.com/semantic-release/git) with a **CI user who had permission to push to master branch** to update `package.json` file.
+
+## Node project configuration
+
+[GitHub Actions](https://github.com/features/actions) supports [Workflows](https://help.github.com/en/articles/configuring-workflows) allowing to test on multiple Node versions and publishing a release only when all test pass.
+
+**Note**: The publish pipeline must run a [Node >= 8 version](../support/FAQ.md#why-does-semantic-release-require-node-version--83).
+
+### `.github/workflows/release.yml` configuration for Node projects
+
+This example is a minimal configuration for [`semantic-release`](https://github.com/semantic-release/semantic-release) with a build running Node 12 when receives a new commit at `master` branch. See [Configuring a workflow](https://help.github.com/en/articles/configuring-a-workflow) for additional configuration options.
+
+```yaml
+name: Release
+on:
+  push:
+    branches:
+      - master
+jobs:
+  install:
+    name: "Generate release"
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@master
+        with:
+          node-version: '12.4.0'
+      - name: Setup NPM Token
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
+      - name: Setup NPM registry
+        run: npm config set registry https://registry.npmjs.org/
```

Comment:
Are you sure the two `Setup NPM` steps are necessary? It works just fine for me without it, e.g. see https://github.com/octokit/core.js/blob/master/.github/workflows/release.yml

Reply:
@gr2m Oh, yes! You are right... I removed it.. please, take a look again.
Thanks!